src/libpriv/rpmostree-postprocess.c

```diff
@@ -992,6 +992,22 @@ rpmostree_postprocess_final (int            rootfs_dfd,
         return glnx_prefix_error (error, "SELinux postprocess");
     }
 
+  /* We keep hitting issues with the ostree-remount preset not being
+   * enabled; let's just do this rather than trying to propagate the
+   * preset everywhere.
+   */
+  static const char preset_dir[] = "usr/lib/systemd/system-preset";
+  if (!glnx_shutil_mkdir_p_at (rootfs_dfd, preset_dir, 0755, cancellable, error))
+    return FALSE;
+  { g_autofree char *preset_path = g_build_filename (preset_dir, "rpm-ostree-auto.preset", NULL);
```

Comment:
I think the convention is `NN-name` -- so maybe... `99-rpm-ostree-auto.preset`? (Earliest one wins -- it seems like if there somehow *is* another earlier preset that does `disable ostree-remount.service`, then there's something wrong with the package set one is trying to compose and we should ideally fail at compose time, but barring that, just fail the boot?).

Reply:
Er...but it's `99-default-disable.preset` that disables everything and at that point we'd be relying on alphabetical ordering...

>  it seems like if there somehow is another earlier preset that does disable ostree-remount.service, then there's something wrong with the package set one is trying to compose and we should ideally fail at compose time, but barring that, just fail the boot?).

Except, argh, how did I manage to omit a numeric prefix?  I'm picking `40` for the fixup :arrow_down: - if anyone for some reason really wanted to disable it they can pick from a pretty early range like `10`.  That said this feels like something we should only care about if someone shows up with a valid use case, I can't think of one.